src/AugmintToken.test.js

```diff
@@ -1,12 +1,14 @@
-require("./utils/env.js")();
+const { loadEnv } = require("./utils");
 const { assert } = require("chai");
 const EthereumConnection = require("./EthereumConnection.js");
 const AugmintToken = require("./AugmintToken.js");
 
 const CCY = "EUR";
 
+const config = loadEnv()
+
 describe("AugmintToken connection", () => {
-    const ethereumConnection = new EthereumConnection();
+    const ethereumConnection = new EthereumConnection(config);
```

Comment:
this way we are passing every env var to the constructor (including other moduls env vars). Might be ok for now but we can think of encapsulating by module somehow and just pass the relevant part? Just dismiss this if you feel it's  over-engineering for now.

Reply:
it feels a a bit over-engineering :) Config can be a bunch of properties, but the receiver object should check if it has every properties what it needs.